### ISSUE TITLE
Option to exclude some sub modules

### ISSUE THREAD

**Issue by cgl** [seddonym/impulse#16]
It would be great to be able to exclude a given list of sub modules from the dependency graph.
This would enable us (1) to find the problematic modules, (2) get a better/simpler picture removing not very critical nodes from the graph.

**Comment by seddonym** [comment]
Agreed. Perhaps the simplest way to do this would be to support passing in a regular expression, something like:

`impulse drawgraph mypackage --exclude=^mypackage\.foo*`

Happy to consider a pull request for this! Or if you don't feel like doing that, it would be pretty easy to write a custom script based on Impulse's `draw_graph` function, [which isn't very complicated](https://github.com/seddonym/impulse/blob/master/src/impulse/application/use_cases.py).
